cluster/ci/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -0,0 +1,48 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ghproxy',
+        rules: [
+          {
+            alert: 'ghproxy-status-code-abnormal-%sXX' % code_prefix,
+            // excludeing 304 because 304 is not expected.
+            // https://developer.github.com/v3/#conditional-requests
+            // excluding 404 because otherwise
+            // "this is going to spam us just about 24/7 with the OWNERS thing"
+            // TODO: Undo 404 after https://jira.coreos.com/browse/DPTP-447 is done
+            expr: |||
+              sum(github_request_duration_count{status=~"%s..", status!="304|404"})/sum(github_request_duration_count{status!="404"}) * 100 > 5
+            ||| % code_prefix,
+            'for': '1m',
```

Comment:
I did not change `1m` to a bigger range because at least from beginning, i want to see how often it fires.
Just use today as an example of our normal days.